include/openssl/bio.h.gen

```diff
@@ -132,19 +129,6 @@ extern "C" {
 
 /* Deliberately outside of OPENSSL_NO_SCTP - used in bss_dgram.c */
 #  define BIO_CTRL_DGRAM_SCTP_SET_IN_HANDSHAKE    50
```

Comment:
This can also go. Nothing uses this and the time where the two values clashed was very limited. Also remove mentions in `bio_dgram.c`, of course.
```suggestion
```